pivorakmeetup/pivorak-web-app
# Show 10 events per page for admin events

Fixed #453 

### Description

Added more convenient pagination for admin/events page 10 entries per page.

### How to test instructions

1. go to /admin/events page verify that there are 25 entries per page
2. switch to local branch "453-show-10-events-per-page-for-admin-events"
4. go to /admin/events page verify that there are 10 entries per page

### Pre-merge checklist
 
- [+ ] The PR relates to a single subject with a clear title and description
- [ +] Verify that feature branch is up-to-date with `development` (if not - rebase it). 

### Screenshots

| Before                                      | After                                       |
| ------------------------------------------- | ------------------------------------------- |
| 
![before_25_per_page](https://user-images.githubusercontent.com/10061671/28307304-29a14f04-6bab-11e7-9b3a-536f61a83e98.png) | ![after_25_per_page](https://user-images.githubusercontent.com/10061671/28307325-35698bd0-6bab-11e7-89fe-0c62a77d46ea.png)
 |
